components/src/core/components/ListTable/ListTable.vue

```diff
@@ -291,6 +295,10 @@ export default defineComponent({
       return props.items;
     });
 
+    const allSelectDisabled = computed(() => {
```

Comment:
this behaviour seems not implemented for cardtable and old list component. please check

Reply:
Added it to the card table as well